cluster/terraform_aks_cluster/analytics.tf

```diff
@@ -84,3 +84,83 @@ resource "azurerm_monitor_data_collection_rule_association" "aks" {
   target_resource_id      = azurerm_kubernetes_cluster.main.id
   data_collection_rule_id = azurerm_monitor_data_collection_rule.aks.id
 }
+
+resource "azurerm_monitor_action_group" "slack" {
+  name                = "${var.resource_prefix}-tsc-${var.environment}-slack"
+  resource_group_name = var.resource_group_name
+  short_name          = substr("tsc-${var.environment}", 0, 12)
+
+  webhook_receiver {
+    name        = "slack"
+    service_uri = data.azurerm_key_vault_secret.slack_webhook.value
```

Comment:
We don't create the action groups in tf. There are commands to add to the Makefile, looks like PR 345 will add them when merged, and have been already run.

Can't send to a slack webhook directly, it needs a logic app or similar. So we just send via email using an azure action group (s189x01-tsc) e.g. https://github.com/DFE-Digital/terraform-modules/blob/main/aks/postgres/resources.tf#L201

Reply:
resolved